admin-dev/themes/default/template/controllers/currencies/conversion_rate.tpl

```diff
@@ -0,0 +1,27 @@
+
+<div class="row">
+	<div class="col-lg-12">
+		{if Module::isInstalled("cronjobs")}
+		<div id="currencyCronjobLiveExchangeRate" class="panel">
+			<div class="panel-heading">
+				{l s='Live exchange rates'}
+				<div class="pull-right checkbox titatoggle unchecked-red checkbox-slider--b-flat">
+					<label>
+						<input type="checkbox" {(0 != Configuration::get('PS_ACTIVE_CRONJOB_EXCHANGE_RATE'))?'checked="checked"':''}><span></span>
+					</label>
+				</div>
+				<div class="clearfix"></div>
+			</div>
+			<span class="status disabled {(0 == Configuration::get('PS_ACTIVE_CRONJOB_EXCHANGE_RATE'))?'':'hide'}">{l s="The exchange rates aren't automaticly updated"}</span>
```

Comment:
Could use an {else} statement to avoid getting info from Configuration class again